src/pages/Register.vue

```diff
@@ -1,24 +1,307 @@
-  </main>
+  <section id="register">
+    <div class="register-container">
+      <div class="register-form" id="register-form">
+        <form method="POST" id="registrationform">
+          <h1>Register</h1>
+          <div class="alert">Message Sent</div>
+          <p>
+            Fill in the required information for you to be showcased on our
+            website.
+          </p>
+          <br />
+          <div class="item">
+            <p>Full Name</p>
+            <input type="text" id="fullName" placeholder="ex. Juan" value="" />
+          </div>
+          <div class="item">
+            <p>Quest Title</p>
+            <select
+              id="quest-title"
+              name="quest-name"
+              v-bind:questTitles="questTitles"
+            >
+              <option disabled selected hidden>Select a quest</option>
+              <option
+                v-for="questTitle in questTitles"
+                v-bind:key="questTitle.id"
+              >
+                {{ questTitle.title }}
+              </option>
+            </select>
+          </div>
+          <div class="item">
+            <p>Date of Completion</p>
+            <input
+              type="date"
+              id="date"
+              placeholder="Select the date of completion"
+              value=""
+            />
+          </div>
+          <div class="item">
+            <p>Display Photo</p>
+            <div class="upload">
+              <input
+                type="file"
+                id="finisher-img"
+                name="finisher-img"
+                accept="image/*"
+              />
+            </div>
+          </div>
+          <div>
+            <button
+              class="submit-btn"
+              id="submit-btn"
+              type="submit"
+              v-on:click="login"
+            >
+              Submit
+            </button>
+          </div>
+        </form>
+      </div>
+      <div class="register-photo">
+        <img :src="require('@/assets/images/register.jpg')" />
+      </div>
+    </div>
+
+    <RegisterModal></RegisterModal>
+  </section>
 </template>
 
 <script>
+import RegisterModal from "../components/Register-Components/RegisterModal.vue";
+
 export default {
-}
+  name: "Register",
+  components: {
+    RegisterModal,
+  },
+
+  data() {
+    return {
+      questTitles: [
+        { id: 1, title: "Baseline Data,ML,AI" },
+        { id: 2, title: "Baseline Infrastructure" },
+        { id: 3, title: "BigQuery Basics for Data Analysis " },
+        { id: 4, title: "Cloud Engineering " },
+        { id: 5, title: "DevOps Essentials " },
+        { id: 6, title: "GCP Essentials " },
+        { id: 7, title: "Google Developer Essentials  " },
+      ],
+    };
+  },
+
+  methods: {
+    login(e) {
+      e.preventDefault();
+      const modal = document.querySelector(".modal");
+      const registerContainer = document.querySelector(".register-container");
+      if (confirm("Confirm?")) {
```

Comment:
also can we disable the submit button and require each input instead of a pop up

Reply:
hi wdym by this, no more register modal?